app/views/idv/by_mail/enter_code_rate_limited/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= render(
+      'idv/shared/error',
+      title: t('idv.gpo.title'),
+      heading: t('idv.failure.gpo.rate_limited.heading'),
+      options: [
+        {
+          url: MarketingSite.contact_url,
+          text: t('idv.troubleshooting.options.contact_support', app_name: APP_NAME),
+          new_tab: true,
+        },
+      ].select(&:present?),
+    ) do %>
+      <p>
+        <%= t(
+              'errors.enter_code.rate_limited_html',
+              timeout: distance_of_time_in_words(
+                Time.zone.now,
+                [@expires_at, Time.zone.now].compact.max,
+                except: :seconds,
+              ),
+            ) %>
+      </p>
+
+      <p>
+        <strong>
+          <% if decorated_sp_session.sp_name.present? %>
+            <%= link_to(
+                  t(
+                    'idv.failure.exit.with_sp',
+                    app_name: APP_NAME, sp_name: decorated_sp_session.sp_name,
+                  ),
+                  return_to_sp_failure_to_proof_path(
+                    step: 'verify_id',
```

Comment:
Is `verify_id` the correct step name here? Same usage below.

Reply:
changed to the right step